testing/tests/DevExpress.ui.widgets.dataGrid/keyboardNavigation.tests.js

```diff
@@ -230,7 +236,20 @@ QUnit.module("Keyboard navigation", {
                 columnHeadersView: new View("columnHeadersView")
             },
             option: function(name) {
-                return that.options[name];
+                const properties = name.split('.'),
+                    options = that.options;
+                if(arguments.length === 1) {
+                    return properties.reduce((prev, cur) => prev && prev[cur], options);
+                } else if(arguments.length === 2) {
+                    const value = arguments[1];
+                    properties.reduce((prev, cur, index) => {
+                        const found = prev && prev[cur];
+                        if(typeUtils.isDefined(found) && index === properties.length - 1) {
+                            prev[cur] = value;
+                        }
+                        return found;
+                    }, options);
+                }
```

Comment:
Complex code in mocked option method. Can we simplify it?

Reply:
This code needs to find nested option like "keyboardNavigation.enable".
In my opinion this code is not so complex thanks to "reduce" function.